test/visibility/import/enablesUnqualified/importAndImport2.good

```diff
@@ -0,0 +1,2 @@
+importAndImport2.chpl:19: In function 'main':
+importAndImport2.chpl:20: error: 'x' undeclared (first use this function)
```

Comment:
Shouldn't `x` be visible due to the `use C` ? Because module C did `import A.x` ? (really `public import A.x;`)?

Reply:
imports are private by default

Reply:
Are there public imports yet?

Reply:
Anyway can you add comments to these to emphasize that they are assuming private import?

Reply:
Engin is adding the ability to declare that in a separate PR (but it won't apply to imports that enable unqualified access)